backend/audit/templates/audit/submission_checklist/submission-checklist.html

```diff
@@ -60,6 +60,18 @@
                         {% endif %}
                     </li>
                     
+                    {% comment %} Tribal data release consent {% endcomment %}
+                    <li class="usa-icon-list__item padding-bottom-105" id="tribal_consent-completed">
+                        {% include './icon-list-icon.html' with completed=False %}
+                        <div>
+                            <p class="text-bold margin-0">
+                                <a class="usa-link"
+                                    href="{% url 'audit:TribalAuditConsent' report_id %}">Tribal data release</a>
+                            </p>
+                            <p class="margin-0">Auditees must opt in or opt out of making their reporting package publicly available.</p>
```

Comment:
@jperson1 Since this will show up as inactive (I assume) when non-tribal orgs are doing submissions, I think the language here should include something about how it only applies to tribal orgs, otherwise non-tribal submitters might think they need to access this form somehow.

Reply:
Ideally:

1. This does not appear *at all* for non-tribal submissions.
2. It should appear inactive as part of tribal submissions, unless...
3. You are visiting the page as the auditee certifying official.

Reply:
Making it entirely hidden requires either adding a fourth state to the logic that drives the page or treating logic for the tribal submission form as entirely different from the other sections. Either are  possible but require more work and I suspect will make test coverage harder.
There are similar concerns for gating it on user role; I don't think we do that elsewhere on the page.

Reply:
I'd like to understand why it's hard, if we can do a short call at some point. But, showing this to 40K users when only 500 need it feels like a bad choice for 99% of our users.

Reply:
Per co-work conversation, the tribal data consent block should not determine its state using `submission_progress_check`, but rather have its own custom logic in the Django template.